catroid/src/test/java/org/catrobat/catroid/test/formulaeditor/InternFormulaStateTest.java

```diff
@@ -58,7 +58,7 @@ public void setUp() throws Exception {
 		internState = new InternFormulaState(internTokenList, null, 0);
 		internStateToCompare = new InternFormulaState(internTokenList, internTokenSelection, 0);
 
-		assertFalse(internState.equals(internStateToCompare));
+		assertNotEquals(internState, internStateToCompare);
```

Comment:
Here too please, sorry that I didn't point it out earlier.
Could you also please rename the "internStateToCompare" to "unexpectedInternState" or something similar.

Reply:
assertNotEquals(unexpected, actual)
But you can also use assertThat (see at the test below), it's a bit more readable than assertNotEquals/assertEquals.

Reply:
If I used assertThat, the line will be like 
		assertThat(internState, not(equalTo(internStateToCompare)));
So I think it's hard to read now